app/helpers/application_helper.rb

```diff
@@ -12,4 +12,8 @@ def input_checked(key, value)
   def facet_has_any_selected_values?(key)
     params.has_key?(key)
   end
+
+  def absolute_url_for(path)
+    URI.join("https://www.gov.uk", path)
```

Comment:
For the sake of documentation, it looks like `:only_path` works when `url_for` is constructing the URL from a hash or object. Giving it just the path throws an error.